### PR TITLE
Added Prims [Java] Closes #116

### LINKED ISSUE #116: Prims Algorithm [Java]
https://en.wikipedia.org/wiki/Prim%27s_algorithm

### PR DESCRIPTION
**Pull request by yashLadha**
Closes #116

#### By submitting this pull request I confirm I've read and complied with the below requirements.

- [x] I have read the [Contribution guidelines](https://github.com/iiitv/algos/blob/master/CONTRIBUTING.md) and I am confident that my PR reflects them.
- [x] I have followed the [coding guidelines](https://github.com/iiitv/algos/blob/master/CONTRIBUTING.md#cs) for this project.
- [x] My code follows the [skeleton code structure](https://github.com/iiitv/algos/blob/master/CONTRIBUTING.md#sample).
- [x] This pull request has a descriptive title. For example, `Added {Algorithm/DS name} [{Language}]`, not `Update README.md` or `Added new code`.

### CODE REVIEW

**Comment by yashLadha** [comment]
@aviaryan @singhpratyush There was a issue raised by codacy bot about class or interface doesn't have a package description. I have just removed the import statements and issue is resolved. How is this so?

**Comment by aviaryan** [comment]
@yashLadha Codacy's check is still pending.

**Comment by yashLadha** [comment]
Sir i had rebase all the commits. It will pass

**Comment by yashLadha** [comment]
What should be the package name? @aviaryan

**Comment by aviaryan** [comment]
@yashLadha No need for package name. If I remember right, other Java codes don't have them.

**Comment by yashLadha** [comment]
So how should i correct it guide me. @aviaryan

**Comment by aviaryan** [comment]
@yashLadha Leave it empty, like https://github.com/iiitv/algos/blob/master/binary_search/BinarySearch.java

**Comment by yashLadha** [comment]
I don't get the difference

**Comment by yashLadha** [comment]
@aviaryan Please review it

**Comment by yashLadha** [comment]
There is no such need for package name in this code, in my opinion. What do u think? @singhpratyush @Monal5031

File: prims/Prims.java [hunk 1]
```diff
@@ -0,0 +1,77 @@
+public class Prims {
+
+    private static class primsMST {
```
Comment (singhpratyush) [inline prims/Prims.java]: Better to use the same class. No one else would be able to use this class.

Comment (aviaryan) [inline prims/Prims.java]: Yes, this sub-class isn't needed.

Comment (yashLadha) [inline prims/Prims.java]: Done :+1:

**Comment by singhpratyush** [comment]
@aviaryan: Take a look.

**Pull request merged**